packages/core/src/layout/ErrorPage/MicDrop.tsx

```diff
@@ -18,16 +18,29 @@ import React from 'react';
 import { makeStyles } from '@material-ui/core';
 import MicDropSvgUrl from './mic-drop.svg';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   micDrop: {
     maxWidth: '60%',
-    bottom: 10,
-    right: 10,
+    bottom: 16,
+    right: 16,
```

Comment:
Ooh this is an opportunity to make these `theme.spacing(2)` instead for consistency

Reply:
Done